docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -49,6 +49,7 @@ Some features are enabled by default. You can disable these feature by setting t
 | `recordedQueriesMulti`               | Enables writing multiple items from a single query within Recorded Queries                                                                                                                                                   | Yes                |
 | `transformationsRedesign`            | Enables the transformations redesign                                                                                                                                                                                         | Yes                |
 | `grafanaAPIServer`                   | Enable Kubernetes API Server for Grafana resources                                                                                                                                                                           | Yes                |
+| `awsAsyncQueryCaching`               | Enable caching for async queries for Redshift and Athena. Requires that the `useCachingService` feature toggle is enabled and the datasource has caching and async query support enabled                                     | Yes                |
```

Comment:
Is it necessary to say `Requires that the toggle is enabled` here, since this PR makes the feature GA?

Reply:
It's a different toggle (that is GA!) but users could disable that one and then be surprised nothing caches so I think it's still useful?

Reply:
oh, I see now :) I misread!